blog/2023-02-22-mysql-s3-blog/index.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Streamlining Your Data Backup Strategy: Setting up a MySQL to S3 Pipeline with Vanus
+title: Streamlining Your Data Backup Strategy Setting up a MySQL to S3 Pipeline with Vanus
```

Comment:
title: Streamlining Your Data Backup Strategy; Setting up a MySQL to S3 Pipeline with Vanus

Reply:
```suggestion
title: Streamlining Your Data Backup Strategy; Setting up a MySQL to S3 Pipeline with Vanus
```